cypress/integration/cerberus/task-management.spec.js

```diff
@@ -0,0 +1,46 @@
+describe('Verify Task Management Page', () => {
+  beforeEach(() => {
+    cy.fixture('users/cypressuser@lodev.xyz.json').then((user) => {
+      cy.login(user.username);
+    });
+  });
+
+  it('Should check all the tabs on task management page', () => {
```

Comment:
The wording could be slightly improved:
```suggestion
  it('Should render all the tabs on task management page', () => {
```

Reply:
will change it.